docs/content/docs/tutorials/aurora-serverless-v2.md

```diff
@@ -137,7 +143,11 @@ kubectl create secret generic -n "${APP_NAMESPACE}" ack-creds \
   --from-literal=password="${RDS_DB_PASSWORD}"
 ```
 
-You can now create an Aurora Serverless v2 cluster:
+
+You can now create an Aurora Serverless v2 cluster for both the PostgreSQL and
+MySQL database engines. The example below uses the PostgreSQL engine. To use
+MySQL, set `ENGINE_TYPE` to `aurora-mysql` and `ENGINE_VERSION` to `8.0`.
```

Comment:
nit: Worth linking to a page where users can find the latest version of the engines, so you aren't forced to update this tutorial with every release?

Reply:
fixed